Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -185,7 +182,7 @@ AP Level: 2
 
 == EAL Tactical Throwing Axe $160 ==
 
-Damage: 25 + 3*STR + DEX + [Thrown Weapon Skill]/2 + 1xBleed
+Damage: 25 + 3*STR + DEX + 1xBleed

```

Comment:
Same here; this was also written before we limited skills to a 6 point/level cap. We need to amp the damage on each of these by a bit.